docs/source/cupy-reference/kernel.rst

```diff
@@ -108,24 +108,32 @@ The ElementwiseKernel class does the indexing with broadcasting automatically, w
 On the other hand, we sometimes want to write a kernel with manual indexing for some arguments.
 We can tell the ElementwiseKernel class to use manual indexing by adding the ``raw`` keyword preceding the type specifier.
 
-We can use the special variables ``n`` and ``i`` for the manual indexing.
-``n`` indicates total number of elements to apply the elementwise operation.
+We can use the special variable ``i`` and ``_ind`` for the manual indexing.
 ``i`` indicates the index within the loop.
+``_ind`` holds the infromation about the shape of the underlying array.
+
+* ``_ind.size()`` returns the number of elements to apply the elementwise operation.
+* ``_ind.shape`` is the array that represents the shape of the array.
```

Comment:
Is this member visible?
`shape_` is private member.
https://github.com/pfnet/chainer/blob/master/cupy/core/carray.cuh#L197